docs/build-hibench.md

```diff
@@ -48,3 +53,19 @@ Supported modules includes: micro, ml(machine learning), sql, websearch, graph,
 For Spark 2.0 and Spark 2.1, we add the benchmark support for Structured Streaming. This is a new module which cannot be compiled in Spark 1.6. And it won't get compiled by default even if you specify the spark version as 2.0 or 2.1. You must explicitly specify it like this:
 
     mvn -Psparkbench -Dmodules -PstructuredStreaming clean package 
+
+### Build using JDK 1.8
+If you are interested in building using Java 11 specify scala, spark and hadoop version as below
+
+    mvn clean package -Dhadoop=3.2 -Dspark=2.4 -Dscala=2.11
```

Comment:
This might be misleading that Java 8 can only be used with the specified Scala/Hadoop/Spark version. I think we can drop this section and only leave `JDK 11` section.

Reply:
No problem.

Reply:
Done